app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,38 @@
+@font-face {
+  font-family: 'ef_cuisinesregular';
+  src: url('efcuisines-regular-webfont.eot');
+  src: url('efcuisines-regular-webfont.eot?#iefix') format('embedded-opentype'),
+  url('efcuisines-regular-webfont.woff') format('woff'),
+  url('efcuisines-regular-webfont.ttf') format('truetype'),
+  url('efcuisines-regular-webfont.svg#ef_cuisinesregular') format('svg');
+  font-weight: normal;
+  font-style: normal;
+
+}
+
+.cuisine-font {
+  font-family: 'ef_cuisinesregular';
+  font-size: 60px;
+  color: brown;
+}
+
+#header {
+  position: relative;
+  width: 100%;
+
+  h1{
+    position: absolute;
+    top: 20%;
+    text-align: center;
+    width: 100%;
+    color: white;
+    line-height:40px;
+  }
+  button{
+    border: 0;
+    position: absolute;
+    top: 5%;
+    right:3%;
```

Comment:
Seems pretty strange, not sure how it supposed to be. Can you attach screenshot?